src/main/java/ui/MenuControl.java

```diff
@@ -87,7 +87,35 @@ private Menu createFileMenu() {
         return file;
 
     }
-    
+
+    private void onBoardNew() {
+        logger.info("Menu: Boards > New");
+
+        if (confirmNewBoardCreation()) {
+            panels.closeAllPanels();
+            onBoardSaveAs();
+        } else {
+            logger.info("New board creation cancelled");
+        }
+    }
+
+    private boolean confirmNewBoardCreation() {
```

Comment:
Write a header comment for the method. Remember to mention the behavior if there are no panels.